bug(android): release notes too long for Play Store

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

Release build 15.0.90 failed for Android because release notes were too long (546 chars instead of 500). We need a way to reduce the release note content (perhaps making it static?)

https://build.palaso.org/viewLog.html?buildId=283126&buildTypeId=KeymanAndroid_Build&tab=buildLog

```
[08:32:13][Step 5/6] Caused by: com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
[08:32:13][Step 5/6] POST https://androidpublisher.googleapis.com/androidpublisher/v3/...
[08:32:13][Step 5/6] {
[08:32:13][Step 5/6]   "code": 403,
[08:32:13][Step 5/6]   "errors": [
[08:32:13][Step 5/6]     {
[08:32:13][Step 5/6]       "domain": "global",
[08:32:13][Step 5/6]       "message": "The release created has notes in language en-US with length 546, which is too long (max: 500).",
[08:32:13][Step 5/6]       "reason": "forbidden"
[08:32:13][Step 5/6]     }
[08:32:13][Step 5/6]   ],
[08:32:13][Step 5/6]   "message": "The release created has notes in language en-US with length 546, which is too long (max: 500).",
[08:32:13][Step 5/6]   "status": "PERMISSION_DENIED"
[08:32:13][Step 5/6] }
```

---

<!-- Delete the irrelevant sections -->

**Keyman for iPhone/iPad/Android:**
 - Device: n/a <!-- e.g. iPhone 6 -->
 - OS: n/a <!-- e.g. iOS 8.1 -->
 - Keyman version: 15.0.90 <!-- e.g. 14.0.192-alpha -->

--- a/android/build-help.sh
+++ b/android/build-help.sh
@@ -117,12 +117,23 @@ if [ $TIER = "stable" ]; then
   PLAY_RELEASE_NOTES="$KEYMAN_ROOT/android/KMAPro/kMAPro/src/main/play/release-notes/en-US/default.txt"
 fi
 echo "Generating Play Store release notes to $PLAY_RELEASE_NOTES"
+echo "" > "$PLAY_RELEASE_NOTES"
 
-# Filtering for lines starting with '*'
-grep '^\s*\*.*$' "$KEYMAN_ROOT/android/help/about/whatsnew.md" > $PLAY_RELEASE_NOTES
-
-# Check release notes are under Play Store limit of 500 characters
-if test "$( wc -m < $PLAY_RELEASE_NOTES )" -gt 500; then
-  warn "Warning: Play Store release notes over 500 characters"
-  echo '* Bug fixes and improvements' > $PLAY_RELEASE_NOTES
-fi
+# Copy whatsnew.md to release notes 1 line at a time,
+# filtering for lines that start with "*".
+# Play Store release notes have a limit of 500 characters
+wc=0
+FILTERED_LINES=`grep '^\s*\*.*$' "$KEYMAN_ROOT/android/help/about/whatsnew.md"`
+IFS=$'\n'      # Change IFS to new line
+for line in $FILTERED_LINES
+do
+  CHARS_IN_RELEASE_NOTES=$( wc -m < $PLAY_RELEASE_NOTES )
+  CHARS_IN_CURRENT_LINE=$( wc -m <<< $line )
+  if (( CHARS_IN_RELEASE_NOTES + CHARS_IN_CURRENT_LINE + 1 < 450 )); then
+    # Copy line to Play Store release notes
+    echo $line >> $PLAY_RELEASE_NOTES
+  else
+    # 450 chars reached
+    break
+  fi  
+done

--- a/android/build-help.sh
+++ b/android/build-help.sh
@@ -117,4 +117,12 @@ if [ $TIER = "stable" ]; then
   PLAY_RELEASE_NOTES="$KEYMAN_ROOT/android/KMAPro/kMAPro/src/main/play/release-notes/en-US/default.txt"
 fi
 echo "Generating Play Store release notes to $PLAY_RELEASE_NOTES"
-cp $KEYMAN_ROOT/android/help/about/whatsnew.md $PLAY_RELEASE_NOTES
+
+# Filtering for lines starting with '*'
+grep '^\s*\*.*$' "$KEYMAN_ROOT/android/help/about/whatsnew.md" > $PLAY_RELEASE_NOTES
+
+# Check release notes are under Play Store limit of 500 characters
+if test "$( wc -m < $PLAY_RELEASE_NOTES )" -gt 500; then
+  warn "Warning: Play Store release notes over 500 characters"
+  echo '* Bug fixes and improvements' > $PLAY_RELEASE_NOTES
+fi

--- a/android/build-help.sh
+++ b/android/build-help.sh
@@ -122,7 +122,6 @@ echo "" > "$PLAY_RELEASE_NOTES"
 # Copy whatsnew.md to release notes 1 line at a time,
 # filtering for lines that start with "*".
 # Play Store release notes have a limit of 500 characters
-wc=0
 FILTERED_LINES=`grep '^\s*\*.*$' "$KEYMAN_ROOT/android/help/about/whatsnew.md"`
 IFS=$'\n'      # Change IFS to new line
 for line in $FILTERED_LINES
@@ -131,9 +130,10 @@ do
   CHARS_IN_CURRENT_LINE=$( wc -m <<< $line )
   if (( CHARS_IN_RELEASE_NOTES + CHARS_IN_CURRENT_LINE + 1 < 450 )); then
     # Copy line to Play Store release notes
-    echo $line >> $PLAY_RELEASE_NOTES
+    echo "$line" >> $PLAY_RELEASE_NOTES
   else
     # 450 chars reached
+    echo '* Additional bug fixes and improvements' >> $PLAY_RELEASE_NOTES
     break
   fi  
 done

--- a/android/build-help.sh
+++ b/android/build-help.sh
@@ -133,6 +133,7 @@ do
     echo "$line" >> $PLAY_RELEASE_NOTES
   else
     # 450 chars reached
+    warn "Warning: Play Store release notes approaching 500 character limit"
     echo '* Additional bug fixes and improvements' >> $PLAY_RELEASE_NOTES
     break
   fi  

